src/puppet-web/firer.ts

```diff
@@ -367,7 +366,7 @@ async function checkRoomTopic(m: Message): Promise<void> {
   if (/^You$/.test(changer) || /^你$/.test(changer)) {
     changerContact = Contact.load(this.userId)
   } else {
-    changerContact = room.member(changer)
+    changerContact = room.member({remark: changer}) || room.member({nick: changer})
```

Comment:
I have two questions:

1. room.member() should use which order？  

I have found the rules of the name display order in wechat group's in #173 as follows:

>  Display order:
 @ Event: `displayName > nickName`
 system message `remark > nickName`
 on-screen Names:`remark > displayName > nickName`

I suggest choosing `on-screen Name`, because it contains `system message` and it is useful in `firer.ts`

2. Now room.member can just return one contact, should it return a contact list?